CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Fixed
+
+- Drop greedy metrics and increase scrape interval to 60s.
```

Comment:
mind splitting into 2 lines?
also I don't think this is a fix but a change, so it should go in the `Changed` section